frontend/src/citizen-frontend/applications/editor/ApplicationFormData.ts

```diff
@@ -281,7 +281,7 @@ export function formDataToApiData(
             street: form.contactInfo.childFutureStreet,
             postalCode: form.contactInfo.childFuturePostalCode,
             postOffice: form.contactInfo.childFuturePostOffice,
-            movingDate: LocalDate.parseFi(form.contactInfo.childMoveDate)
+            movingDate: LocalDate.parseFiOrNull(form.contactInfo.childMoveDate)
```

Comment:
these are also just hiding a problem. the validation function needs to ensure that if there is a future address then moving date has valid value